org.eclipse.jdt.core/model/org/eclipse/jdt/core/IClasspathAttributeDelta.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Simeon Andreev and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.jdt.core;
+
+/**
+ * A delta which describes changes in {@link IClasspathAttribute} of a classpath
+ * between two discrete points in time.  Given a delta,
+ * clients can access the name of the affected attribute
+ * as well as the current value of the attribute.
+ * <p>
+ * Deltas have a different status depending on the kind of change they represent.
+ * The list below summarizes each status (as returned by {@link #getKind})
+ * and its meaning (see individual constants for a more detailed description):
+ * <ul>
+ * <li>{@link #ADDED} - The attribute has been added.</li>
+ * <li>{@link #REMOVED} - The attribute has been removed.</li>
+ * <li>{@link #CHANGED} - The value of the attribute has been changed.
+ * </ul>
+ *
+ * @since 3.33
```

Comment:
```suggestion
 * @since 3.32
```

Reply:
Is this still required?

Reply:
No, this isn't needed to be changed now.